libnopegl/src/params.h

```diff
@@ -135,6 +135,14 @@ struct ngl_node;
  */
 #define NGLI_PARAM_FLAG_ALLOW_NODE (1U<<4)
 
+/*
+ * Imply that the parameter refers to an external path.
+ *
+ * Only applicable to string parameters. It is also not possible to live change
+ * such a parameter.
```

Comment:
Can this restriction be easily lifted at some point ? I can totally see changing medias on the fly while not having to reload an entire graph for that.

Reply:
Maybe but it requires building a relationship from the node parameter back to the scene arrays.

Reply:
Restriction removed and live change added for medias